### ISSUE TITLE
TF protobuf exported from tutorial page introduces constructs not accepted by MIGraphX

### ISSUE THREAD

**Issue by mvermeulen** [ROCmSoftwarePlatform/AMDMIGraphX#261]
Tensorflow tutorial page https://www.tensorflow.org/tutorials has a simple MNIST example that can be frozen to a TF protobuf using the following additional code -
```
sess = tf.Session()
sess.run(tf.global_variables_initializer())
graph_def = tf.get_default_graph().as_graph_def()
output_graph = tf.graph_util.convert_variables_to_constants(sess,graph_def,['dense_1/Softmax'])
with tf.gfile.GFile('frozen_mnist.pb','wb') as f:
   f.write(output._graph.SerializeToString())
```

Unfortunately the frozen graph introduces constructs that can not be read as inputs into MIGraphX - so we need to decide what to do for them.  Perhaps document, perhaps work around, etc.  This PR documents what is found in this particular example so we can see how to address them.

The most significant seems to be that the Keras dropout layer introduces a mess of different operators.  This can be seen by comparing the results of the "summarize_graph" tool without Dropout layer
```
mev@chilecito:~/source/tensorflow/tensorflow-cpu$ ./bazel-bin/tensorflow/tools/graph_transforms/summarize_graph --in_graph=/home/mev/tf-save/frozen_tmnist.pb 
Found 1 possible inputs: (name=flatten_input, type=float(1), shape=[?,28,28]) 
No variables spotted.
Found 1 possible outputs: (name=dense_1/Softmax, op=Softmax) 
Found 407054 (407.05k) const parameters, 0 (0) variable parameters, and 0 control_edges
Op types used: 8 Const, 4 Identity, 2 BiasAdd, 2 MatMul, 1 Pack, 1 Placeholder, 1 Relu, 1 Reshape, 1 Shape, 1 Softmax, 1 StridedSlice
To use with tensorflow/tools/benchmark:benchmark_model try these arguments:
bazel run tensorflow/tools/benchmark:benchmark_model -- --graph=/home/mev/tf-save/frozen_tmnist.pb --show_flops --input_layer=flatten_input --input_layer_type=float --input_layer_shape=-1,28,28 --output_layer=dense_1/Softmax
```
and with the Dropout layer
```
mev@chilecito:~/source/tensorflow/tensorflow-cpu$ ./bazel-bin/tensorflow/tools/graph_transforms/summarize_graph --in_graph=/home/mev/tf-save/frozen_tmnist.pb 
Found 1 possible inputs: (name=flatten_input, type=float(1), shape=[?,28,28]) 
No variables spotted.
Found 1 possible outputs: (name=dense_1/Softmax, op=Softmax) 
Found 407059 (407.06k) const parameters, 0 (0) variable parameters, and 4 control_edges
Op types used: 13 Const, 7 Identity, 3 Switch, 2 BiasAdd, 2 Sub, 2 Shape, 2 Mul, 2 MatMul, 2 Add, 1 Merge, 1 Pack, 1 Placeholder, 1 PlaceholderWithDefault, 1 RandomUniform, 1 RealDiv, 1 Relu, 1 Reshape, 1 Softmax, 1 StridedSlice, 1 Floor
To use with tensorflow/tools/benchmark:benchmark_model try these arguments:
bazel run tensorflow/tools/benchmark:benchmark_model -- --graph=/home/mev/tf-save/frozen_tmnist.pb --show_flops --input_layer=flatten_input --input_layer_type=float --input_layer_shape=-1,28,28 --output_layer=dense_1/Softmax
```
The second additional issue can also be seen in the output above where the input shape is given a "-1,28,28" indicating it works for multiple batch sizes.

**Comment by mvermeulen** [comment]
Newer version of tutorial uses estimators, use this instead (MEV to verify and then close issue).

**Comment by mvermeulen** [comment]
Works with newer tutorial format.
